scss/components/button-group.scss

```diff
@@ -51,3 +51,17 @@ $block-alt: #{$fd-namespace}-segmented-button;
     }
   }
 }
+.fd-button-group .fd-button--compact:hover,
+.fd-button-group .fd-button:hover {
+  &.is-disabled {
+    --fd-button-background-color: white;
+    @include fd-var-color("background-color", white);
+  }
+}
+
+.fd-button-group .fd-button--compact:focus,
+.fd-button-group .fd-button:focus {
+  &.is-disabled {
```

Comment:
I don't think disabled should be implemented as a CSS class (like `is-disabled`).  Although it can be used for styling, it provides no a11y support.  When going through the playground page with a screen reader, the disabled buttons sound just like the enabled buttons.  A disabled button should actually remove itself from the tab order.

I know this isn't unique to buttons or button groups, but the elements should be getting the `disabled` attribute (for elements that support it) and then we can use _that_ in our styling selectors.

Reply:
I would go further and suggest we *strip* `&.is-disabled` styles from the CSS. 

Using it will consistently lead to bad a11y practices.

Reply:
@droshev